spezi-module/onboarding/src/main/java/edu/stanford/spezikt/spezi_module/onboarding/invitation/InvitationCodeUiState.kt

```diff
@@ -0,0 +1,16 @@
+package edu.stanford.spezikt.spezi_module.onboarding.invitation
+
+data class InvitationCodeUiState(
+    val invitationCode: String = "",
+    val error: String? = ""
+)
+
+enum class TextFieldType {
```

Comment:
This would not make much sense if we just have a single text field in the screen